lib/auth/init.go

```diff
@@ -1269,6 +1275,20 @@ func createPresetDatabaseObjectImportRule(ctx context.Context, rules services.Da
 	return nil
 }
 
+// createPresetHealthCheckConfig creates a default preset health check config
+// resource that enables health checks on all resources.
+func createPresetHealthCheckConfig(ctx context.Context, svc services.HealthCheckConfig) error {
+	preset := services.NewPresetHealthCheckConfig()
```

Comment:
should we also list other health check configs here in case user has deleted the default but created others?